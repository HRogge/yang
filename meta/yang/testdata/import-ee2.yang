module import-ee2 {
    prefix "";
    namespace "";
    revision 0;

    leaf z {
        type string;
    }
}