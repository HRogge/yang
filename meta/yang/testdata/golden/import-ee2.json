{
"module":{
  "ident":"import-ee2",
  "revision":{
    "rev-date":"0"},
  "definitions":[
    {
      "ident":"z",
      "leaf":{
        "ident":"z",
        "type":{
          "ident":"string"}}}]}}